modules/fscloud/variables.tf

```diff
@@ -203,6 +203,14 @@ variable "operating_system" {
   type        = string
   description = "The operating system of the workers in the default worker pool. If no value is specified, the current default version OS will be used. See https://cloud.ibm.com/docs/openshift?topic=openshift-openshift_versions#openshift_versions_available ."
   default     = null
+  validation {
+    condition = anytrue([
+      var.operating_system == null,
+      var.operating_system == "RHEL 8",
```

Comment:
Is `"RHEL 8"` really a valid value (which the space in it?). Is it not supposed to be `REDHAT_8_64` ?

Reply:
Thanks for the info @ocofaigh. I am verifying this as I was having some confusion. Thought of checking both names.

This confusion was due to two values/names provided in the docs: 
<img width="420" alt="image" src="https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/assets/78964991/2c92745e-ec5b-4181-a702-652287f9354e">

However while creating clusters, we see the name as below (one name i.e `RHEL 8` also mentioned in the docs):
<img width="314" alt="image" src="https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/assets/78964991/3d927e51-c1bc-4a44-b992-df895527abef">